### ISSUE TITLE
LEDs don't work

### ISSUE THREAD

**Issue by DieHertz** [jamesturton/bose-qc35-usb-c#11]
Hi, thank you for your effort and for the schematics!

I had asked my friend from China to produce a few boards according to the schematic and BOM, and have successfully installed one today. All of the buttons work, charging works, but none of the LEDs come up, all the boards I have are like that.

I know it's probably some fault on my side, but maybe you've got an idea what could be wrong or how to troubleshoot.

**Comment by jamesturton** [comment]
Hi @DieHertz!
The first thing I would do would be to check polarity of the LEDs you are using. Sometimes they have the marking on the anode and sometimes on the cathode.

**Comment by DieHertz** [comment]
Haven't gotten to my multimeter yet, but took some photos in the meantime. I wonder if there's something missing, e.g. that 3 pin transistor next to LEDs.

Update:
Q1 seems to be missing, apart from that it's only the resistors`R1, R2, R3, R4, R17, R18, R19` which are without value according to BOM.

![IMG_20240306_090827](https://github.com/jamesturton/bose-qc35-usb-c/assets/1427681/1691ee37-62c4-4272-bdb5-c8819e971888)
![IMG_20240306_090805](https://github.com/jamesturton/bose-qc35-usb-c/assets/1427681/17fb6dd7-5e57-4cbb-ba18-ccbfd5884a07)

**Comment by DieHertz** [comment]
Finally saw what my guy did wrong - he did not read the instructions on Q1 and U1. I removed some optional components and it still does not work. Back to the bench I guess :-D

After poking it with a multimeter I found out that all of the LEDs have wrong polarity and also wrong colors.
Replaced all of the, now the LEDs work, but:
1. Blue LED is always on, blinking when bluetooth pairing is on, logic seems to be inverted for it?
2. When plugging in USB green and blue LEDs blink rapidly and charging does not start

**Comment by DieHertz** [comment]
Somehow the ~~fuse~~ ferrite bead was causing issues with charging (it looked like the headphones were alternating between charging and not charging by rapidly blinking the green LED).
I have discovered so by elimination, when everything else failed, I tried supplying external 5V directly to VCC and it started charging. Then I tried shorting the fuse and it also started charging.

That's weird, no idea what can be wrong with the ~~fuse~~ ferrite bead, but it's a correct 330R one according to my multimeter. All 5 boards behave like that, shorting the fuse making them work properly for charging.

Hmm I just realized that maybe the 
~~fuse~~ ferrite bead was actually a 330 Ohm resistor 😅
